python/core/qgsmargins.sip

```diff
@@ -138,90 +138,14 @@ class QgsMargins
 
 
 bool operator==( const QgsMargins &lhs, const QgsMargins &rhs );
-%Docstring
- Returns \c true if ``lhs`` and ``rhs`` are equal; otherwise returns \c false.
- :rtype: bool
-%End
```

Comment:
if it pass, do we merge as is and I'll have a look to the 3 remaining files later on?